packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TableActionHeader.tsx

```diff
@@ -76,8 +75,7 @@ export const TableActionHeader = ({
   return (
     <div
       className={twMerge(
-        'sticky left-0 top-0 flex w-auto flex-nowrap items-center justify-between gap-4 bg-white p-3 text-sm',
```

Comment:
Making top edges rounded as it won't be dependent on `TableSentOrReceivedFundsSwitch` anymore